src/controller/expensesController.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const { expensesServices } = require('../services/expensesServices');
+const { usersServices } = require('../services/usersServices');
+
+const getAll = (req, res) => {
+  const expenses = expensesServices.getAllExpenses(
+    {
+      ...req.query,
+      userId: +req.query.userId,
+    }
+  );
+
+  res.send(expenses);
+};
+
+const addExpense = (req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  if (
+    !usersServices.getUserById(userId)
+    || isNaN(Date.parse(spentAt))
+    || !title
+    || !(typeof amount === 'number')
+    || !category
+    || !note

```

Comment:
if you have several checks will be better to move the result of them in separate boolean variable